docs/plugins.asciidoc

```diff
@@ -43,10 +37,11 @@ You can specify URLs that use the HTTP, HTTPS, or `file` protocols.
 [float]
 === Installing Plugins to an Arbitrary Directory
 
-Use the `-d` or `--plugin-dir` option to specify a directory for plugins, as in the following example:
+Use the `-d` or `--plugin-dir` option after the `install` command to specify a directory for plugins, as in the following 
+example:
 
 [source,shell]
-bin/kibana plugin -i elasticsearch/sample-plugin/latest -d <path/to/directory>
+$ bin/kibana-plugin install elasticsearch/sample-plugin/latest -d <path/to/directory>
 Installing sample-plugin
 Attempting to extract from https://download.elastic.co/elasticsearch/sample-plugin/sample-plugin-latest.tar.gz
```

Comment:
The plugin url will now include the exact version number of Kibana in its url, so this link should reflect that.  The format will be: `https://download.elastic.co/kibana/${plugin}/${plugin}-${version}.zip`